automatic/apache-httpd/tools/helpers.ps1

```diff
@@ -1,10 +1,53 @@
+
```

Comment:
Fix code style and identation and remove extra empty lines, to be like the rest of the code.